kit/cli/loglevelflag.go

```diff
@@ -0,0 +1,42 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/spf13/pflag"
+	"go.uber.org/zap/zapcore"
+)
+
+type levelValue zapcore.Level
```

Comment:
This boilerplate file is pretty much identical to `kit/cli/idflag.go`, but with `zapcore.Level` instead of `influxdb.ID`.